### ISSUE TITLE
Staging Review finding: Buttons with opposite purposes have the same `aria-label`

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#88449]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Appointments
**Product:** Appointments
**Feature:** Appointment Details Redesign

## Findings details
**VA.gov Experience Standard - issue:** [User can't perceive an element.](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

When cancelling a pending appointment, two buttons appear on the page: "Yes, cancel request" and "No, do not cancel." Both of these buttons have the same `aria-label`, "Cancel appointment," meaning both will be announced to screen reader users as "Cancel appointment." Screen reader users won't be able to distinguish between them, meaning:
1. If they do want to cancel their appointment, they have to get lucky and pick the correct button.
2. If they don't want to cancel their appointment, they don't have a perceivable route to do that.

<details><summary>Link, screenshot or steps to recreate</summary>

This is on the page that's displayed when you select a pending appointment and then click "Cancel request." The two buttons are coded as:

```
<button type="button" aria-label="Cancel appointment">Yes, cancel request</button>
```
and
```
<button type="button" aria-label="Cancel appointment" class="usa-button-secondary">No, do not cancel</button>
```

</details>

## Recommended action

I'm guessing this is a copy/paste error. For these two buttons, the visible text label is fine --- no need for an `aria-label` at all. My recommendation would be to remove the `aria-label` entirely on both.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 4.1.2 Name, Role, Value - Level A](https://www.w3.org/TR/WCAG21/#name-role-value)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [x] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [x] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by outerpress** [comment]
@ldelacosta I'm good with Brian's rec to remove the aria labels from the buttons he noted.

**Comment by outerpress** [comment]
@ldelacosta I'm good with Brian's rec to remove the aria labels from the buttons he noted.

**Comment by outerpress** [comment]
@ldelacosta I'm good with Brian's rec to remove the aria labels from the buttons he noted.

**Comment by ldelacosta** [comment]
Appointments FE ticket for reference [#88523](https://github.com/department-of-veterans-affairs/va.gov-team/issues/88523)

**Comment by ldelacosta** [comment]
@shiragoodman - We addressed the ticket in [88523](https://app.zenhub.com/workspaces/appointments-team-603fdef281af6500110a1691/issues/gh/department-of-veterans-affairs/va.gov-team/88523). Let me know if you find additional issues. Thanks!

**Comment by shiragoodman** [comment]
@briandeconinck FYI

@ldelacosta do you want our help validating the fix?

**Comment by ldelacosta** [comment]
@shiragoodman - we have validated on our end. If you want to double check, feel free.

**Comment by outerpress** [comment]
This one is complete, we removed the aria labels. Closing (please reopen if I missed anything!)

<img width="1640" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMUJrQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--a66c0db71736d3d2470676325c8ebd8b149c8dd6/Screenshot%202024-08-16%20at%208.27.29%E2%80%AFAM.png" alt="Screenshot 2024-08-16 at 8.27.29 AM.png" />
